src/components/Weather.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+
+const WeatherComponent = () => {
+  const [weatherData, setWeatherData] = useState(null);
+
+  useEffect(() => {
+    const API_KEY = 'b71ca2424f7ca2f981190ce292e41ccd';
+    const city = 'Tel Aviv';
+    const apiUrl = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${API_KEY}&units=metric`;
```

Comment:
those are constants, use const naming convention for all of them, and place them in settings file of the component